### ISSUE TITLE
Unable to perform version check.

### ISSUE THREAD

**Issue by Beyarz** [geany/geany-plugins#761]
Unable to perform version check.
Error code: 6
Error message: »Unacceptable TLS certificate«

Here's the debug:
19:57:29: Updatechecker MESSAGE	: Checking for updates (querying URL "http://geany.org/service/version.php")
19:57:30: Updatechecker WARNING	: Connection error: Code: 6; Message: Unacceptable TLS certificate

**Comment by eht16** [comment]
I assume you are using Windows?
If so, try to create a new file in `C:\Program Files (x86)\Geany\ssl\certs\ca-bundle.crt` with the contents in https://pastebin.geany.org/eR622/, restart Geany and try again (obviously adjust the Geany installation path if it differs).

It might help to restart Geany after creating the file.
The pasted CA bundle only contains one of the Let's Encrypt intermediate certificates which is necessary for a proper TLS connection.

If it works then (it did on my Windows test system), then we will ship this certificate with future releases).

**Comment by Beyarz** [comment]
I'm on macOS :)

**Comment by eht16** [comment]
Ok, not sure where to place the file then.
Is there some global or system-wide way to add CA bundles?

If not, we probably need to ship the bundle for macOS as well after we found out where to place it :).

**Comment by Beyarz** [comment]
I tried to import the certificate to the keychain access app but without any success.

**Comment by techee** [comment]
No idea really (see #752).

**Comment by Beyarz** [comment]
So I assume the issue is not with Geany?

**Comment by techee** [comment]
No, it's not Geany itself (works under Linux), the problem is with the dependencies of Geany and their packaging on macOS.

**Comment by Beyarz** [comment]
I see, so what can we do in order to solve this?

**Comment by techee** [comment]
Well, one can play with gtk-osx packages, try updating to latest releases of libraries, fix compilation problems, check if all the files are bundled correctly... which I'm not going to do myself (no fun). But it's also possible someone already did it in the meantime and that the next time I make a release these things will be fixed.

**Comment by Beyarz** [comment]
I'll give it a try!

**Comment by techee** [comment]
@Beyarz You are a hero then! You can find everything related to Geany packaging here:

https://github.com/geany/geany-osx
